tests/SerializationTest.php

```diff
@@ -164,4 +168,24 @@ public function testDatesInBodyParametersAreSerializedInZuluFormat(): void
 
         $this->assertEquals('2024-01-01T00:00:00Z', $body['packageDetail']['shipDate']);
     }
+
+    public function testDeserializeDateTimeWithTimezone(): void
+    {
+        $now = new DateTime();
+        $now->setTimeZone(new DateTimeZone('+02:00'));
+
+        $result = GetMyFeesEstimateResponse::deserialize([
+            'payload' => [
+                'FeesEstimateResult' => [
+                    'status' => 'Success',
+                    'FeesEstimate' => [
+                        'TimeOfFeesEstimation' => $now->format(DateTimeInterface::ATOM),
+                    ],
+                ],
+            ],
+        ]);
+        $this->assertNotNull($result);
+        $this->assertInstanceOf(DateTimeInterface::class, $result->payload->feesEstimateResult->feesEstimate->timeOfFeesEstimation);
+        $this->assertEquals( (new DateTimeZone('+02:00'), $result->payload->feesEstimateResult->feesEstimate->timeOfFeesEstimation->getTimeZone() );
```

Comment:
Looks like you have a syntax error here. If you run the tests with `composer test` before you push, you'll know if they're working or not.

Please run `composer test` and `composer lint` before you commit and push, so that you know everything is working and formatted properly.